respy/likelihood.py

```diff
@@ -317,14 +321,16 @@ def _internal_log_like_obs(
 
         type_probabilities = special.softmax(np.column_stack(z), axis=1)
 
+        type_probabilities = np.clip(type_probabilities, 1 / MAX_FLOAT, MAX_FLOAT)
```

Comment:
Why do we need the upper bound? It should never be larger than one, right?

Reply:
You are right. Removed it.